src/Cake.Common/Tools/DotNet/MSBuild/MSBuildArgumentBuilderExtensions.cs

```diff
@@ -203,6 +203,12 @@ public static void AppendMSBuildSettings(this ProcessArgumentBuilder builder, Do
                 msBuilder.AppendMSBuildSwitch("property", $"ContinuousIntegrationBuild={continuousIntegrationBuild}");
             }
 
+            // Re-use of MSBuild nodes?
+            if (settings.NodeReuse != null)
+            {
+                msBuilder.Append(string.Concat("/nr:", settings.NodeReuse.Value ? "true" : "false"));
+            }
```

Comment:
```suggestion
            // Re-use of MSBuild nodes?
            if (settings.NodeReuse.HasValue)
            {
                msBuilder.Append(string.Concat("/nodeReuse:", settings.NodeReuse.Value ? "true" : "false"));
            }
```

Let's use the longer name version to make it easier to understand